### DISCUSSION TITLE
Creating sample projects not working in CDTCloudBlueprint

### DISCUSSION THREAD

**Discussion by vinodappu** [eclipse-cdt-cloud/cdt-cloud#29] [General]
![image](https://user-images.githubusercontent.com/65697683/217627412-331cb0b5-d323-4a16-ac7b-0ba07358d19b.png)

Installed CDTCloudBlueprint desktop application, opened and tried the options in Getting Started page, None of the items seem working for me. I'm expecting files in the EXPLORER view.

Let me know how to proceed further.

@planger

**Comment by planger** [comment]
Thanks for raising this issue! I re-tested current `master` and [latest release](https://download.eclipse.org/theia/cdt-cloud/latest/) and couldn't reproduce the issue, unfortunately.

1. Do you have any errors in the console if you look at _Help_ -> _Toggle Developer Tools_ and go to tab _Console_?
2. Are there any errors in the std out? If you run CDT Cloud Blueprint from console you should see output.
3. Which operating system are you using? I only tested this on Linux for now. Maybe this is something Windows-specific?

Thanks!

**Reply by vinodappu** [reply]
Issue is in Windows 10. When I tried debug, I've seen it goes in the marked line. But the path you can see in the watch exists.

![image](https://user-images.githubusercontent.com/65697683/217671149-65968a14-2cd2-4f51-9665-5d115bf06d2a.png)

**Reply by planger** [reply]
Just to make sure I understand, in your case the error in line 33 is thrown, even though the path actually exists?
If yes, this is likely a Windows/Linux path issue. We'll have to debug this in Windows and probably shoudl switch to `fsPath` from Theia, now as we've upgraded to the new version of Theia, anyway.
I created https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/issues/49 to track this issue.

**Reply by vinodappu** [reply]
Yes, the path is actually existing, I see.

**Reply by planger** [reply]
I've opened a PR (https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/pull/50) that hopefully fixes the issue. It'd be great if you could test this PR on your machine. Thank you!

**Reply by vinodappu** [reply]
I've tested, it's working fine!

**Reply by planger** [reply]
Excellent, thanks for testing!

**Comment by sdirix** [comment]
Hi @vinodappu,

The example projects are only created when you have an open folder or workspace. Is that the case for you? You can use `File > Open Folder...` to select one.

We should prompt an error message in case no folder or workspace is open. I now created a ticket for that, see [here](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/issues/48).

**Reply by vinodappu** [reply]
Folder is opened, the issue I've seen in Windows 10.

![image](https://user-images.githubusercontent.com/65697683/217669974-5b9757b5-2d37-4a6b-a856-bd215c833cc1.png)
